rooms/windyTylne.cpp

```diff
@@ -1,24 +1,215 @@
+                std::cout << "Naciskasz przycisk przywołania windy i w tym "
+                             "momencie śmiertelnie razi Cię prąd"
+                          << std::endl;
+                wait_s(1);
+                gameData.alive = false;
+                return ZEWNATRZ;
+              }
+
+              std::cout << "Ślepy, czy głupi? Popsuta jest." << std::endl;
+              wait_s(1);
+              i_c++;
+            } else {
+              std::cout << "No debil no" << std::endl << std::endl;
+            }
+          } while (wybor[0] != '1' && wybor[0] != '2');
+        } else {
+          return WINDA_PARTER;
+        }
+
+      } else if (wybor[0] == '2') {
+        return WINDA_PIETRO;
+
+      } else if (wybor[0] == '3') {
+        return BIBLIOTEKA;
+
+      } else if (wybor[0] == '4') {
+        return GARAZ;
+
+      } else if (wybor[0] == '5') {
+        return KORYTARZ;
+
+      } else {  // TODO(seqre): Add waving "hand"
+        std::cout << std::endl
+                  << std::endl
+                  << "To nie jest miejsce, którego szukasz" << std::endl
+                  << "*Obi-Wan Kenobi's hand wave*" << std::endl
+                  << "Wybierz raz jeszcze..." << std::endl
+                  << "...tym razem mądrze, jeśli można." << std::endl
+                  << std::endl;
+      }
+    } else {
+      input_prompt(wybor);
+      unsigned seed1, seed2;
+      if (wybor[0] == '1') {
+        if (!gameData.winda_repaired) {
+          std::cout << "Jesteś debil, bo zamiast uciekać sprawdzasz windę"
+                    << std::endl;
+          gameData.alive = false;
+          return ZEWNATRZ;
+        }
+        if (rand_r(&seed1) % 10 > 0) {
+          std::cout << "Widzisz jak w ostatniej chwili drzwi od windy "
+                       "zamykają się przed zaioncem"
+                    << std::endl;
+          return WINDA_PARTER;
+        }
+        std::cout << "Niestety, ale zaionc był szybszy od windy" << std::endl;
+        gameData.alive = false;
+        return ZEWNATRZ;
+      }
+      if (wybor[0] == '2') {
+        if (rand_r(&seed2) % 10 > 0) {
+          std::cout << "Widzisz jak w ostatniej chwili drzwi od windy zamykają "
+                       "się przed zaioncem"
+                    << std::endl;
+          return WINDA_PIETRO;
+        }
+        std::cout << "Niestety, ale zaionc był szybszy od windy" << std::endl;
+        gameData.alive = false;
+        return ZEWNATRZ;
+      } else if (wybor[0] == '3') {
+        return BIBLIOTEKA;
+
+      } else if (wybor[0] == '4') {
+        return GARAZ;
+
+      } else if (wybor[0] == '5') {
+        return KORYTARZ;
+
+      } else {
+        std::cout << "Twoje niezdecydowanie działa na korzyść zaionca"
+                  << std::endl;
+        gameData.alive = false;
+        return ZEWNATRZ;
+      }
+    }
   }
-  return WINDY_TYLNE;
+}
+
+Room windyTylne(GameData& gameData) {
+  start(gameData);
+
+  srand(time(nullptr));
```

Comment:
To powinno być w funkcji `main`

Reply:
Masz na myśli całą funkcję wybór_działania czy co?

Reply:
Nie - tylko inicjalizację generatora liczb losowych. Tego typu rzeczy powinny być wykonywane przez kod główny gry.